modules/agar/src/main/ts/ephox/agar/assertions/ApproxStructures.ts

```diff
@@ -257,6 +304,22 @@ const assertStyles = (expectedStyles: Record<string, StringAssert>, actual: Suga
   });
 };
 
+const assertExactMatchStyles = (expectedStyles: Record<string, StringAssert>, actual: SugarElement<Element>) => {
+  const allDefinedStyles = Obj.keys(expectedStyles);
+  const actualDefinedStyles = Obj.keys(Css.getAllRaw(actual));
+
+  const isEqual = assertEqualArray(
+    allDefinedStyles,
+    actualDefinedStyles
+  );
+
+  if (!isEqual) {
+    throw new Error('Style names were not matching. actual: ' + actualDefinedStyles.join(', ') + ', expected: ' + allDefinedStyles.join(', '));
```

Comment:
Would it be clearer to have wrapping `[]` in the string to better distinguish between actual and expected values?
It might actually be better to use a template string
```suggestion
    throw new Error(`Style names were not matching. actual: [${actualDefinedStyles.join(', ')}], expected: [{allDefinedStyles.join(', '))}]`;
```

Reply:
Yep, I applied the suggestion, clearer to read the actual and expected values now.